python/cuml/cuml/accel/tests/scikit-learn/xfail-list.yaml

```diff
@@ -1155,6 +1074,11 @@
   - "sklearn.manifold.tests.test_spectral_embedding::test_pipeline_spectral_clustering"
   - "sklearn.svm.tests.test_svm::test_custom_kernel_not_array_input[SVR]"
   - "sklearn.svm.tests.test_svm::test_custom_kernel_not_array_input[SVC]"
+- reason: Test crashes with cuml.accel
+  marker: cuml_accel_crashes
+  run: false
+  tests:
+  - "sklearn.tests.test_common::test_estimators[RandomForestClassifier()-check_classifiers_regression_target]"
```

Comment:
This test leads to a complete crash, which means the normal `xfail` mechanism can't work. I've added support for `run=False` (https://docs.pytest.org/en/stable/how-to/skipping.html#run-parameter) and marked the test appropriately. I'll add a follow-up issue to fix the crash later.

Reply:
That's very concerning. Would appreciate if you could create the issue right away.